REFERENCE.md

```diff
@@ -760,6 +782,89 @@ Number of seconds to wait for Deployment and StatefulSet rollouts to complete.
 
 ## Plans
 
+### <a name="pam_toolsinstall_chart"></a>`pam_tools::install_chart`
+
+Install a helm chart.
+
+Optionally adds a helm *repository_uri* before installing.
+
+Optionally waits for rollout of the installed chart based on a
+*part_of* selector.
+
+#### Parameters
+
+The following parameters are available in the `pam_tools::install_chart` plan:
+
+* [`targets`](#targets)
+* [`chart_name`](#chart_name)
+* [`release`](#release)
+* [`repository_uri`](#repository_uri)
+* [`values_yaml`](#values_yaml)
+* [`namespace`](#namespace)
```

Comment:
Similar to above, this link seems to go to the `namespace` under "helm_install_chart".